packages/components/src/EmptyState/EmptyState.module.scss

```diff
@@ -3,21 +3,22 @@
 @import "~@kaizen/design-tokens/sass/border";
 @import "./mixins";
 
+$border-width: var(--border-width-1);
```

Comment:
Isn't this a bit unnecessary?

Reply:
Hmm true. I want a scoped variable for calculations, and sometimes they ended up in different classes. But if not, should be doable in CSS :) Will play around with it